olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationClient.h

```diff
@@ -148,6 +148,54 @@ class AUTHENTICATION_API AuthenticationClient {
     unsigned int expires_in{0};
   };
 
+  /**
+   * @brief The Apple sign-in properties structure.
+   */
+  struct AppleSignInProperties {
```

Comment:
Let's change this awful approach and not do a nested struct. Please move to a separate file.